src/hash.cr

```diff
@@ -699,10 +793,31 @@ class Hash(K, V)
     false
   end
 
+  # Returns *true* if a `Hash` has any key-value pair.
   def any?
     !empty?
   end
 
+  # Iterates over each key-value pair with the given block, 
+  # replacing *memo* with the returned value each time. 
+  # Returns *memo* after all iteration.
```

Comment:
Mention the argument more explicitly, specifically that it provides the initial value?

> Returns the last value of \*memo\*.

?